mwdb/schema/remotes.py

```diff
@@ -3,3 +3,7 @@
 
 class RemotesListResponseSchema(Schema):
     remotes = fields.List(fields.Str())
+
+
+class RemoteShareRequestSchema(Schema):
```

Comment:
Maybe `Options` instead of `Share` so it can be extended in the future without naming change
```suggestion
class RemoteOptionsRequestSchema(Schema):
```